mftf/2.3/metadata.md

```diff
@@ -405,34 +422,35 @@ Root element that points to the corresponding XML Schema.
+| `name`          | string                                                                       | optional | Name of the operation.                                                                                                                       |
+| `dataType`      | string                                                                       | required | Data type of the operation. It refers to a `type` attribute of data entity that will be used as a source of input data.                      |
+| `type`          | Possible values: `create`, `delete`, `update`, `get`.                        | required | Type of operation.                                                                                                                           |
+| \*`url`         | string                                                                       | optional | A routing URL of the operation. Example value: `"/V1/categories"`.                                                                           |
+| \*\*`auth`      | Possible values: `adminOath`, `adminFormKey`, `customerFormKey`, `anonymous` | optional | Type of authorization of the operation.                                                                                                      |
+| `method`        | string                                                                       | optional | HTTP method of the operation. Possible values: `POST`, `DELETE`, `PUT`, `GET`.                                                               |
+| `successRegex`  | string                                                                       | optional | Determines if the operation was successful. Parses the HTML body in response and asserts if the value assigned to the `successRegex` exists. |
+| `returnRegex`   | string                                                                       | optional | Determines if the response contains the matching value to return.                                                                            |
+| `removeBackend` | boolean                                                                      | optional | Removes backend name from requested URL. Applicable when `auth="adminFormKey"`.                                                              |
+| `filename`      | string                                                                       | optional |                                                                                                                                              |
+
+- \*`url` - full URL is a concatenation of _ENV.baseUrl_ + `/rest/` + _url_.
+  To reuse data of a required entity or returned response use a field key wrapped in curly braces such as `{sku}`.
+  When the data to reuse is of a different type, declare also the type of data such as `{product.sku}`.
+  Examples: `"/V1/products/{sku}/media/{id}"`.
```

Comment:
missing type? should be
Examples: `"/V1/products/{product.sku}/media/{id}"`.

Reply:
:ok: